db/migrate/20201125105949_add_standars_product_was_testing_against_to_tests.rb

```diff
@@ -0,0 +1,8 @@
+class AddStandarsProductWasTestingAgainstToTests < ActiveRecord::Migration[6.0]
```

Comment:
Please change: typos:
```suggestion
class AddStandardsProductWasTestedAgainstToTests < ActiveRecord::Migration[6.0]
```